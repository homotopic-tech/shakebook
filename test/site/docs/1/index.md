---
title: The First Subsection
description: The First Subsection
---

Lorem ipsum
