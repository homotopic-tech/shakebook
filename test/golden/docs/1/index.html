<html>
  <head>
    <title>The First Subsection</title>
  </head>
  <body>
    <h1>This post page is called The First Subsection</h1>
    <p>It was generated from the source docs/1/index.md</p>
    <p>For the url /docs/1/index.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the subsections:
      <ul>
      </ul>
    </p>
  </body>
</html>
